from exceptions.BaseException import BaseException

class UsedEmailException(BaseException):
    pass